src/applications/find-forms/actions/index.js

```diff
@@ -62,10 +63,15 @@ export const fetchFormsThunk = (query, options = {}) => async dispatch => {
 
   try {
     // Attempt to make the API request to retreive forms.
-    const results = await fetchFormsApi(query, { mockRequest });
+    const resultsDetails = await fetchFormsApi(query, { mockRequest });
 
     // If we are here, the API request succeeded.
-    dispatch(fetchFormsSuccess(results));
+    dispatch(
+      fetchFormsSuccess(
+        resultsDetails.results,
+        resultsDetails.allFormsTombstone,
```

Comment:
Random question, but what does allFormsTombstone mean? 🙂

Reply:
Ah nvm! You're helpful comment informed me! 🙂 https://github.com/department-of-veterans-affairs/vets-website/pull/15499/files#diff-e4d8eff49920c28f2d11d038d9c5196f38bb5e1c9eef2377b9656fb3bbdc1693R34